shiny/ui/__init__.py

```diff
@@ -31,8 +31,12 @@
     tags,
 )
 
-# Expose the following modules for extended usage: ex: ui.fill.as_fill_item(x)
-from . import css, fill  # noqa: F401  # pyright: ignore[reportUnusedImport]
+from . import (
+    # The css module is for internal use, so we won't re-export it.
+    css,  # noqa: F401  # pyright: ignore[reportUnusedImport]
+    # Expose the fill module for extended usage: ex: ui.fill.as_fill_item(x).
+    fill,
+)
```

Comment:
Oh, these might need to be imported separately?

Reply:
Good catch, thanks.